action.yaml

```
@@ -37,7 +37,7 @@ runs:
     - name: Setup node
       uses: actions/setup-node@v3
       with:
-        node-version-file: .nvmrc
+        node-version: "18.17.0"
```

What about using `github.action_path`?

```suggestion
        node-version: ${{ github.action_path }}/.nvmrc
```

---

Maybe for another time…